test/unit/app/presenters/publishing_api/worldwide_office_presenter_test.rb

```diff
@@ -5,7 +5,7 @@ def present(...)
     PublishingApi::WorldwideOfficePresenter.new(...)
   end
 
-  test "presents a Worldwide Office ready for adding to the publishing API" do
+  test "presents a Worldwide Office belonging to a non-editionable worldwid organisation ready for adding to the publishing API" do
```

Comment:
typo here in `worldwid[e]`, and the test below

Reply:
Thanks.  Have adding the missing 'e' in both places.